modules/serverless-functions-on-cluster-builds.adoc

```diff
@@ -27,14 +27,21 @@ include::snippets/technology-preview.adoc[leveloffset=+1]
 +
 [source,terminal]
 ----
```

Comment:
wrt buildpack we should mention it is not supported -- pure upstream

Reply:
@matejvasek We removed the buildpacks step from here and put it inside a note that specifically states that they are not supported:

https://github.com/openshift/openshift-docs/pull/51030/commits/f4b292d06d1640408cebee3541be1cb8cf2fe4cf#diff-3f93bad3a6dccebfacff22389de39b99cf910accf4cadd053414fd97349accd0R49

I suppose you didn't notice this. If not, feel free to reopen this.